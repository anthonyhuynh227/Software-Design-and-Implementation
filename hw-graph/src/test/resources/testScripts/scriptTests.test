#Testing file

#Create graph
CreateGraph A
CreateGraph B
CreateGraph C

#Add nodes into graphs
AddNode B b1
AddNode C c1
AddNode C c2

#Add edges into graphs
AddEdge C c1 c2 e1
AddEdge C c2 c1 e2

#Print all nodes in the graph
ListNodes A
ListNodes B
ListNodes C

#Print all child nodes of a node with outgoing edge
ListChildren C c1
ListChildren C c2