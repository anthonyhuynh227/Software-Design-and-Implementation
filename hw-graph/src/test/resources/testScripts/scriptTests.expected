#Testing file

#Create graph
created graph A
created graph B
created graph C

#Add nodes into graphs
added node b1 to B
added node c1 to C
added node c2 to C

#Add edges into graphs
added edge e1 from c1 to c2 in C
added edge e2 from c2 to c1 in C

#Print all nodes in the graph
A contains:
B contains: b1
C contains: c1 c2

#Print all child nodes of a node with outgoing edge
the children of c1 in C are: c2(e1)
the children of c2 in C are: c1(e2)